src/ert/_c_wrappers/enkf/model_config.py

```diff
@@ -40,11 +40,21 @@ def __init__(
             else self.DEFAULT_HISTORY_SOURCE
         )
 
-        self.runpath_format_string = (
-            runpath_format_string
-            if runpath_format_string is not None
-            else self.DEFAULT_RUNPATH
-        )
+        if runpath_format_string is None:
+            self.runpath_format_string = self.DEFAULT_RUNPATH
+        elif "%d" in runpath_format_string:
+            self.runpath_format_string = runpath_format_string
+            logger.warning(
+                "RUNPATH keyword should use syntax "
+                f"`{self.DEFAULT_RUNPATH}` "
+                "instead of deprecated syntax "
+                f"`{runpath_format_string}`"
+            )
+        elif not any(x in runpath_format_string for x in ["<ITER>, <IENS>"]):
+            self.runpath_format_string = runpath_format_string
+            logger.error(
+                "RUNPATH keyword shall use syntax " f"`{self.DEFAULT_RUNPATH}`."
```

Comment:
Maybe `should` instead instead of `shall` ?

Reply:
Sure thing.

And added new issue for suggester ui.
https://github.com/equinor/ert/issues/4543